test/kitchen/site-cookbooks/dd-agent-disable-system-repos/recipes/default.rb

```diff
@@ -13,6 +13,6 @@
   end
 elsif ['suse', 'opensuseleap'].include?(node[:platform])
   execute 'disable all zypper repositories' do
-    command 'zypper mr -da'
+    command 'ZYPP_LOCK_TIMEOUT=120 zypper mr -da'
```

Comment:
Could you add a comment to explain why we are setting this?

Reply:
done